From fba740a5662539938f4f825347912521b6b506d8 Mon Sep 17 00:00:00 2001
From: =?utf-8?q?R=C3=A9mi=20Cardona?= <remi@gentoo.org>
Date: Fri, 21 Nov 2008 16:39:00 +0100
Subject: [PATCH 04/42] xfree86: don't reset Auto(Add|Enable)Devices, use defaults from xf86Globals

Without this, commit ace38fafb062372dcd3d56378b5b8f86525c6241 is useless
when HAL support is disabled.
---
 hw/xfree86/common/xf86Config.c |    2 --
 1 files changed, 0 insertions(+), 2 deletions(-)

diff --git a/hw/xfree86/common/xf86Config.c b/hw/xfree86/common/xf86Config.c
index e91eadd..b6b4203 100644
--- a/hw/xfree86/common/xf86Config.c
+++ b/hw/xfree86/common/xf86Config.c
@@ -922,7 +922,6 @@ configServerFlags(XF86ConfFlagsPtr flagsconf, XF86OptionPtr layoutopts)
         from = X_CONFIG;
     }
     else {
-        xf86Info.autoAddDevices = TRUE;
         from = X_DEFAULT;
     }
     xf86Msg(from, "%sutomatically adding devices\n",
@@ -934,7 +933,6 @@ configServerFlags(XF86ConfFlagsPtr flagsconf, XF86OptionPtr layoutopts)
         from = X_CONFIG;
     }
     else {
-        xf86Info.autoEnableDevices = TRUE;
         from = X_DEFAULT;
     }
     xf86Msg(from, "%sutomatically enabling devices\n",
-- 
1.6.0.4

